spec/models/answer_spec.rb

```diff
@@ -0,0 +1,165 @@
+require 'rails_helper'
+
+describe Answer do
+  let(:questionnaire) {create(:questionnaire,:id => 1)}
```

Comment:
It is overkilled to use `create` in unit tests, using `double` or `build` instead.